### PR TITLE
added validation for Input of type number

### PR DESCRIPTION
**Pull request by lbelinsk**

### CODE REVIEW

File: src/Input/Input.js [hunk 1]
```diff
@@ -171,6 +171,14 @@ class Input extends Component {
       this.props.onEscapePressed && this.props.onEscapePressed();
     }
   }
+
+  _onChange(e) {
+    if (this.props.type === 'number' && !(/^\d+$/.test(e.target.value))) {
+      return;
```
Comment (mastertheblaster) [inline src/Input/Input.js]: @lbelinsk What about empty values? Guess they should be allowed for number input too?

Comment (lbelinsk) [inline src/Input/Input.js]: right.. i will change the + to *

Comment (lbelinsk) [inline src/Input/Input.js]: Actually it doesn't matter... it only controls if onchange will get triggered.. if nothing is typed it won't even try to trigger it..

Comment (mastertheblaster) [inline src/Input/Input.js]: Let's say i have a number inside the input `1` and i hit backspace button. Will it be ok?

Comment (lbelinsk) [inline src/Input/Input.js]: nice catch... for some reason it is working, but I will change + to * since it is more correct

**Pull request merged**